ublox/src/ubx_packets/types.rs

```diff
@@ -83,8 +83,8 @@ impl<'a> From<&NavVelNedRef<'a>> for Velocity {
 impl<'a> From<&NavPvtRef<'a>> for Position {
     fn from(packet: &NavPvtRef<'a>) -> Self {
         Position {
-            lon: packet.lon_degrees(),
-            lat: packet.lat_degrees(),
+            lon: packet.longitude(),
+            lat: packet.latitude(),
```

Comment:
as yeas go by, I find it very useful to have explicit unit in those applications, as it seriously reduces the risk of mistakes

Reply:
I agree in principle but what could you reasonably expect it to be other than degrees? If we want this kind of safety we should definitely use the newtype pattern to prevent mixing up units. It's something that we could retrofit as well.

Reply:
I will revert this change for now and we can tackle this another time.

In principle I also prefer static types as for example using newtype or agree to use SI units or representative units for the quantity at hand. I also wouldn't expect lat/lon to be anything else then degrees in a public API.